tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -828,6 +828,33 @@ void ROOT::Experimental::Internal::RMiniFileReader::ReadBuffer(void *buffer, siz
 
 ////////////////////////////////////////////////////////////////////////////////
 
+auto &kBlobKeyLen = ROOT::Experimental::Internal::RNTupleFileWriter::kBlobKeyLen;
```

Comment:
Why is this an `auto &`?

Reply:
IIRC this was what the compiler suggested. Do we want to have this a `static constexpr auto kBlobKeyLen =`?

Reply:
I changed the declaration accordingly, as discussed

Reply:
The new version seems to have replace the alias with a hard coded number.  Does it matter if they (inadvertently) get out of sync?

Reply:
It's not clear to me what you are referring to: the new version is
```c++
static constexpr auto kBlobKeyLen = ROOT::Experimental::Internal::RNTupleFileWriter::kBlobKeyLen;
```
which itself is defined as
```c++
static constexpr std::size_t kBlobKeyLen = 42;
```
So the variables are `constexpr` and can never change its value. The value 42 itself is checked with two `R__ASSERT` in `PrepareBlobKey`.

Reply:
My bad. I kept not find the `static constexpr auto kBlobKeyLen` in the diff and mistakenly assumed that `kBlobKeyLen = 42` was new and that somewhere else there was another (pre-existing) `kBlobKeyLen = 42`.

So nevermind, this is resolved.